### ISSUE TITLE
SystemNavigation compatibility issue

### ISSUE THREAD

**Issue by JurajKubelka** [dionisiydk/Calypso#112]
`SystemNavigation` calls `browser spawnHierarchyForClass: aClass selector: aSymbolOrNil` that is not implemented on `ClySystemBrowser` instance side.

```Smalltalk
SystemNavigation >> browseHierarchy: aClass selector: aSelector
	"Open a browser"

	aClass isNil
		ifTrue: [ ^ self ].
	^ Smalltalk tools browser new
		spawnHierarchyForClass: aClass selector: aSelector;
		yourself
```

It is very likely an incorrect solution, at least `Smalltalk tools browser new` should be replaced by ` `Smalltalk tools browser`, I think. for the compatibility reasons, I propose 

```Smalltalk
ClySystemBrowser  >> spawnHierarchyForClass: aClass selector: aSymbolOrNil
	navigationEnvironment ifNil: [ self showEnvironment: ClySystemNavigationEnvironment currentImage ].
	self open.
	aSymbolOrNil 
		ifNil: [ self selectClass: aClass ]
		ifNotNil: [ self selectMethod: (aClass lookupSelector: aSymbolOrNil) ]
```

**Comment by JurajKubelka** [comment]
Steps to reproduce (in Pharo 6.1): Inspect any morph using halo, and choose "browse class morph"

**Comment by dionisiydk** [comment]
Thank's. I added it to upcoming version

**Comment by dionisiydk** [comment]
Fixed in 0.8
